packages/cli/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Created a centralized header class to be used across the files SDK with `ZosFilesHeaders.ts`. [#2436](https://github.com/zowe/zowe-cli/pull/2436)
```

Comment:
We should say what `ZosFilesHeaders.ts` is: " ... across the files SDK with the **file/database/function/whatever** `ZosFilesHeaders.ts`."